.gitignore

```
@@ -10,3 +10,15 @@ node_modules/
 !.yarn/releases
 !.yarn/sdks
 !.yarn/versions
+
+# docs
+_build/
+bin/
+build/
+include/
+lib/
+lib64/
+local/
+share/python-wheels
```

Redundant to next entry?

```suggestion
share/python-wheels
```